package.json

```diff
@@ -8,7 +8,7 @@
     "language server",
     "scalameta"
   ],
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
Bumped this because I would like to release a new version right away, to ensure a smooth upgrade to v0.5.0